### ISSUE TITLE
Publishing

### ISSUE THREAD

**Issue by dkhamsing** [iOS-Goodies/iOS-Goodies#555]
Hello all, first thanks for all the time you put into the issues. It's a really fun and I hope very helpful open-source project.

Out of hundreds of issues, it was very rare that we did not publish timely each week 🎉 

Now I don't know all the details that go into publishing but I think @tiagoalmeida is the main person doing it.

Would it be helpful to have a backup person in case Tiago is unavailable?

@RuiAAPeres 
@tiagoalmeida 
@mariusc 
@rbarbosa 
and many people I forget (feel free to cc)

Please 👍 or 👎 this and we can go from there..

Thx!

**Comment by dkhamsing** [comment]
I guess those interested in helping with publishing should contact @tiagoalmeida / @RuiAAPeres for details

Thank you!

**Comment by tiagoalmeida** [comment]
No one contacted me yet :(

**Comment by dkhamsing** [comment]
hey @mariusc <nudge>

**Comment by mariusc** [comment]
Hey :). Yeah, I didn't contact anyone directly because I thought the discussion would just continue here, after the upvoting/downvoting period of the initial message has finished. Anyway..

I think it would be good to have more persons who could take care of publishing because this way we could probably be more constant and on time with each weekly issue. 
On the other hand, curating the articles for the newsletter is a much bigger responsibility than just suggesting articles on GitHub, and in the end, it's a matter of personal opinion and it's very subjective.

All things considered, I would gladly help with the publishing if both @tiagoalmeida and @RuiAAPeres are unavailable, but if there's one person who I really think deserves to be a curator, that's @rbarbosa. (I mean, seriously, have you seen the contributors graphs? Impressive 👍 )

TL;DR: If you need help with publishing, I can gladly do it. Also, consider Rui Barbosa for this if he wants, because he's been doing this for a longer time than I have :)
